src/main/java/org/spongepowered/api/event/entity/player/PlayerEvent.java

```diff
@@ -38,18 +39,9 @@
      *
      * @return The {@link Player} involved
      */
-    Player getPlayer();
-
-    @Override
-    Player getHuman();
-
-    @Override
-    Player getLiving();
-
     @Override
     Player getEntity();
 
     @Override
-    Player getUser();
-
+    User getUser();
```

Comment:
User Event returns a user, thus the subclasses should as well. Plus if we return a direct player the method is basically pointless and redundant.